state-chain/pallets/cf-pools/src/lib.rs

```diff
@@ -1953,11 +1966,9 @@ impl<T: Config> Pallet<T> {
 		amount_change: IncreaseOrDecrease<AssetAmount>,
 	) -> DispatchResult {
 		let collected_fees: AssetAmount = collected.fees.try_into()?;
-		T::LpBalance::try_credit_account(
-			lp,
-			asset_pair.assets()[(!order.to_sold_side()).into()],
-			collected_fees,
-		)?;
+		let asset = asset_pair.assets()[(!order.to_sold_side()).into()];
```

Comment:
You need to add the recording of fees into `inner_update_range_order` too.

Reply:
I think I do this.